irohad/ametsuchi/temporary_wsv.hpp

```diff
@@ -51,7 +51,7 @@ namespace iroha {
        */
       virtual bool apply(const model::Transaction &transaction,
                          std::function<bool(const model::Transaction &,
-                                            CommandExecutor &, WsvQuery &)>
+                                            WsvCommand &, WsvQuery &)>
                              function) = 0;
```

Comment:
No, both `execute` and `validate` can be called in both `TemporaryWsv` and `MutableStorage`. But in `MutableStorage` we will likely not call `validate`.

`validate` and `execute` are methods of commands, like `class AddPeer : public Command{}` So I don't understand the question